Appsilon/rhino
# Add `test_e2e()`

### Changes
Closes #30. Built on top of #45 to limit the number of conflicts - please merge it first.

### How to test
Initialize a fresh project and run `rhino::test_e2e()`. Cypress should start and run successfully.
